### PR TITLE
Pipeline deployment definition is unclear

### PR DESCRIPTION
**Pull request by BarnyShergold**
## Purpose of this pull request

Pipline deployment definition is unclear #7802

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/config-guide/deployment/pipeline/

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by BarnyShergold** [comment]
I'm a bit unsure as to why I've got a conflict here - can anyone check and see if I shouldn't just resolve?

File: src/_includes/config/split-deploy/split-deploy-overview.md [hunk 1]
```diff
@@ -22,15 +22,4 @@ Staging system
...
-   -  stores
-   -  store views
-   -  products
-   -  product view settings
-   -  catalog
-   -  categories
-   -  category view settings.
-
- You should make all those types of changes in your development system.
+: Your live store. You should make minimal direct configuration changes here and certainly nothing that has not been tested on a Staging instance. If possible configuration changes should be made using [Data Patches](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/declarative-schema/data-patches.html) that have been tested on a Staging/Development instance.
```
Comment (keharper) [inline src/_includes/config/split-deploy/split-deploy-overview.md:25]:
```suggestion
: Your live store. You should make minimal direct configuration changes here, and certainly nothing that has not been tested on a Staging instance. If possible, configuration changes should be made using [Data Patches]({{page.baseurl}}/extension-dev-guide/declarative-schema/data-patches.html) that have been tested on a Staging/Development instance.
```

File: src/cloud/reference/cloud-composer.md [hunk 2]
```diff
@@ -44,14 +44,22 @@ The metapackage depends on the appropriate versions of `vendor/magento/ece-patch
 
 This package depends on a floating version of `vendor/magento/magento-cloud-configuration` (abbreviated _MCC_). It depends on the major and minor version of MCC that correspond to the specified {{site.data.var.ee}} version, and floats on the patch version so that compatible updates to this packages can be automatically pulled by running `composer update`.
 
-## vendor/magento/ece-tools {#ece-tools}
+The main components of this metapackage are :
```
Comment (keharper) [inline src/cloud/reference/cloud-composer.md:47]: Your changes from #7874 have bled into this PR.

**Review by keharper — CHANGES_REQUESTED** [review]
@BarnyShergold  I don't know why you're getting a conflict, but maybe it's related to material from #7874 showing up here.

File: src/_includes/config/split-deploy/split-deploy-overview.md (hunk 1 above)
Comment (BarnyShergold) [reply src/_includes/config/split-deploy/split-deploy-overview.md:25]: Yes like that scan better and how did that URL sneak in?

File: src/cloud/reference/cloud-composer.md (hunk 2 above)
Comment (BarnyShergold) [reply src/cloud/reference/cloud-composer.md]: Nice spot - thanks - all fixed

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Pull request merged**